docs/network/bridge/README.md

```diff
@@ -1,40 +1,57 @@
 # Network Bridge Device Plugin
 
-This device plugin allows user to attach pods to bridges running on nodes.
+This device plugin allows user to attach pods to existing bridges on nodes.
 
 ## Usage
 
-User must specify list of bridges that should be exposed to pods. Please note,
-that bridges must be manually precreated on all nodes.
+The user starts by ensuring that a bridg eexists on a given host.
+If it doesn't exit, then he can easily create it as a root as follows:
```

Comment:
This sentence sounds weird. I'd prefer

"If the bridge doesn't exist, the user can create it (with root privileges) as follows:"